src/components/Breadcrumbs/Breadcrumbs.tsx

```diff
@@ -16,48 +19,135 @@ export interface BreadcrumbItem {
 export interface BreadcrumbProps extends Pick<LinkProps, 'as'> {
   /** A list of `BreadcrumbsItem` objects ( `{href,children}` ) that will construct the Breadcrumb */
   items: BreadcrumbItem[];
+  /** Truncates component if breadcrumb list is too long.
+   * At least 4 breadcrumbs must exist for truncation to occur.
+   */
+  truncate?: boolean;
```

Comment:
since this is defined at pounce level (which means that it's generic), can we make it more dynamic? Could we rename it to `maxDisplayedItems` with a default value of `Infinity` (meaning "no truncation")? 

Then if you say `maxDisplayedItems={5}` you would make sure that you truncate accordingly 😉

Reply:
i would assume we would prefer for the truncation to be set and not configurable so it is consistent across the application usage